packages/form-js-editor/src/features/properties-panel/entries/TextEntry.js

```diff
@@ -15,19 +15,19 @@ export default function TextEntry(props) {
     type
   } = field;
 
-  const entries = [];
+  if (type !== 'text') {
+    return [];
+  }
 
-  if (type === 'text') {
-    entries.push({
+  return [
+    {
       id: 'text',
       component: Text,
       editField: editField,
       field: field,
       isEdited: isTextAreaEntryEdited
-    });
-  }
-
-  return entries;
+    }
+  ];
```

Comment:
If we have such huge PRs, maybe let's try to split up such changes from the very beginning (e.g. by having separate commits from the very start). It's hard to follow what's related otherwise.

Reply:
You're not wrong, I'll try and do this more consistently in the future :D